lib/local-storage-adapter.js

```diff
@@ -20,7 +19,7 @@ function createLocalStorageAdapter (queueName) {
   var setPending = false
   var queueCache = []
 
-  var flushOnDefer = _.debounce(flush, 0)
+  var flushOnDefer = onceOnDefer(flush)
```

Comment:
oohh. I was confused by the first implementation. You had the behavior of `_.once` wrapped in a `_.debounce` which gelled with the name `onceOnDefer`, so I was wondering why the change.